examples/surface_texture_and_colors.py

```diff
@@ -0,0 +1,56 @@
+"""
+Surface with texture and vertex_colors
+==========
+
+Display a 3D surface with texture mapping and colors
+
+.. tags:: visualization-nD
```

Comment:
I've already said it but I'll say again that I love this example for our gallery. 😃